openapi.json

```diff
@@ -2995,6 +2999,66 @@
         }
       }
     },
+    "/v1/passport/users": {
```

Comment:
We might be missing something. here's the error I got when from swagger
![Screenshot 2023-03-10 at 14 52 25](https://user-images.githubusercontent.com/3668156/224219683-a411b400-4696-4643-8e46-88ca0742cb61.png)